frontend/public/models/yaml-templates.ts

```diff
@@ -1074,6 +1074,38 @@ metadata:
 spec:
   hrefTemplate: 'https://example.com/logs?resourceName=\${resourceName}&containerName=\${containerName}&resourceNamespace=\${resourceNamespace}&podLabels=\${podLabels}'
   text: Example Logs
+`,
+  )
+  .setIn(
+    [referenceForModel(k8sModels.ConsoleYAMLSampleModel), 'default'],
+    `
+apiVersion: console.openshift.io/v1
+kind: ConsoleYAMLSample
+metadata:
+  name: example
+spec:
+  apiVersion: batch/v1
```

Comment:
I don't have a great suggestion here, but I worry that having `apiVersion` repeated will be confusing. Maybe we want something like this to make it more clear.

```yaml
spec:
  targetResource:
    apiVersion: batch/v1
    kind: Job
  yaml: ...
```

Reply:
Makes sensto to me :+1

Reply:
So do we want to go with @spadgett's proposal for `targetResource`?